rotkehlchen/chain/ethereum/oracles/uniswap.py

```diff
@@ -252,6 +279,20 @@ def get_price(
         price = FVal(reduce(mul, [item.price for item in prices_and_tokens], 1))
         return Price(price)
 
+    def _resolve_assets(self, assets: tuple[Asset, Asset]) -> list[AssetWithOracles]:
+        """Ensure assets are resolved to AssetWithOracles."""
+        return [asset.resolve_to_asset_with_oracles() for asset in assets]
```

Comment:
nitpick: Hmm okay so this is now used in only 1 place and it's just an 1liner. Just remove the function.